scripts/simple_sensors_capture.py

```diff
@@ -1,22 +1,28 @@
 import time
+from bson import json_util
 
 from peas.sensors import ArduinoSerialMonitor
 
 
-def main(loop=True, delay=1., verbose=False):
+def main(loop=True, delay=1., filename=None, use_mongo=True, send_message=True, verbose=False):
     # Weather object
     monitor = ArduinoSerialMonitor(auto_detect=False)
 
     while True:
-        data = monitor.capture()
+        data = monitor.capture(use_mongo=use_mongo, send_message=send_message)
 
-        if verbose and len(data.keys()) > 0:
-            print(data)
+        if len(data.keys()) > 0:
+            if filename is not None:
+                with open(filename, 'a') as f:
```

Comment:
Seems odd to open the file on each capture. How about open once and flush after each write or in a finally block so that it happens even if there is an exception?

Reply:
Good call, will change.